wicker/core/storage.py

```diff
@@ -463,17 +487,23 @@ class S3PathFactory(WickerPathFactory):
                     - part-1-attempt-2345.parquet
     """
 
-    def __init__(self, s3_root_path: Optional[str] = None) -> None:
+    def __init__(self, s3_root_path: Optional[str] = None, prefix_replace_path: str = "") -> None:
         """Init S3PathFactory.
 
         Args:
             s3_root_path (Optional[str], optional): path to s3 root path. Defaults to None.
+            prefix_replace_pathm (str, optional): path to replace for given prefix. In this class this is generally
```

Comment:
@pickles-bread-and-butter nit prefix_replace_pathm -> prefix_replace_path

Reply:
Done